services/tenant-ui/frontend/src/router/index.ts

```diff
@@ -21,4 +21,16 @@ const router = createRouter({
   routes,
 });
 
+/**
+ * Global router guard to set the page title.
+ * Override only if in the non-default tenant UI.
+ * In this case, the innkeeper UI is the non-default.
+ * TBD: Could move this into a Helm chart value.
+ */
+router.afterEach((to) => {
+  if (to.meta.isInnkeeper === true) {
+    document.title = 'Traction Innkeeper Console';
```

Comment:
let's make this an env var -> config like the tenant side.
see and `UX_APP_TITLE` and `document.title = config.value.frontend.ux.appTitle;`

we can split the work, if you get it into the application code and the scripts/docker-compose. i can run with getting the helm charts and CICD done

Reply:
OK.... Think I've got this working.
I tried being ambitions and updated one of the Helm files..... Caused the pipeline to explode.... Changed it back. I'll let @usingtechnology do the Helm stuff.